main.py

```diff
@@ -0,0 +1,79 @@
+'''Задание
+
+Необходимо создать API для управления списком задач. 
+Каждая задача должна содержать заголовок и описание. 
+Для каждой задачи должна быть возможность указать статус (выполнена/не выполнена).
+
+API должен содержать следующие конечные точки:
+— GET /tasks — возвращает список всех задач.
+— GET /tasks/{id} — возвращает задачу с указанным идентификатором.
+— POST /tasks — добавляет новую задачу.
+— PUT /tasks/{id} — обновляет задачу с указанным идентификатором.
+— DELETE /tasks/{id} — удаляет задачу с указанным идентификатором.
+
+Для каждой конечной точки необходимо проводить валидацию данных запроса и ответа. Для этого использовать библиотеку Pydantic.'''
+
+from fastapi import FastAPI
+import logging
+from pydantic import BaseModel
+
+
+
+app = FastAPI()
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+task_list = []
+
+class Task(BaseModel):
+    id: int
+    title: str
+    description: str
+    status: str
+    
+
+
+@app.get("/tasks")
```

Comment:
Можно указать модель ответа.
```suggestion
@app.get("/tasks", response_model=List[Task])
```